app/views/publish/courses/course_button_panel/_published.html.erb

```diff
@@ -0,0 +1,9 @@
+<span class="govuk-body" data-qa="course__is_findable">
+  <%= course.on_find_panel %>
+</span>
+<span class="govuk-body" data-qa="course__has_vacancies">
+  <%= course.vacancies_panel %>
```

Comment:
you may as well move the `govuk_link_to` to here

Reply:
Good idea, sorted

> ![image](https://user-images.githubusercontent.com/470137/177757967-43dc820f-6c97-43ef-9a04-1a9b4793b703.png)

Good spot, it's being removed